### ISSUE TITLE
Wrong owner on document root

### ISSUE THREAD

**Issue by Letterus** [YunoHost-Apps/my_capsule_ynh#11]
This is wrong: https://github.com/YunoHost-Apps/my_capsule_ynh/blob/master/scripts/install#L170C1-L170C30

Owner needs to be root having permission not more than 5 on groups to make SFTP work. Otherways clients will be disconnected immediately.

**Comment by xmgz** [comment]
thank you @Letterus for opening this issue 


please, have you any workaround to solve it in the mean time? thank you

(I haven't :( )

**Comment by Letterus** [comment]
> thank you @Letterus for opening this issue
> 
> please, have you any workaround to solve it in the mean time? thank you
> 
> (I haven't :( )

@xmgz Yes, you need to set owner to root and permissions to 755 or something manually. But this will only last to the next update, so we need a fix on this.
